draft-ietf-babel-information-model.md

```diff
@@ -170,28 +180,52 @@ The Information Model is hierarchically structured as follows:
 
 
 ~~~~
-information object
-   includes implementation version, router id, this node seqno,
-     enable flag parameters, supported security mechanisms
-   constants object (exactly one per information object)
-      includes UDP port and optional multicast group
-        parameters
-   interfaces object
-      includes interface reference, Hello seqno and intervals,
-        update interval, link type, metric computation parameters
-      neighbors object
-         includes neighbor IP address, Hello history, cost
-           parameters
-      security object (per interface)
-         includes enable flag, self credentials (credential
-           object), trusted credentials (credential object)
-   security object (common to all interfaces)
-      includes enable flag, self credentials (credential
-        object), trusted credentials (credential object)
-   routes object
-      includes route prefix, source router, reference to
-        advertising neighbor, metric, sequence number, whether
-        route is feasible, whether route is selected
++-- babel-information
+   +-- babel-implementation-version
+   +-- babel-enable
+   +-- router-id
+   +-- babel-supported-link-types
+   +-- self-seqno
+   +-- babel-metric-comp-algorithms
+   +-- babel-message-log-enable
+   +-- babel-message-log
+   +-- babel-constants
+   |  +-- babel-udp-port
+   |  +-- babel-mcast-group
+   +-- babel-interfaces
+   |  +-- babel-interface-reference
+   |  +-- babel-interface-enable
+   |  +-- babel-link-type
+   |  +-- babel-interface-metric-algorithm
+   |  +-- babel-mcast-hello-seqno
+   |  +-- babel-mcast-hello-interval
+   |  +-- babel-update-interval
+   |  +-- babel-neighbors
+   |  |  +-- babel-neighbor-address
+   |  |  +-- babel-hello-mcast-history
+   |  |  +-- babel-hello-ucast-history
+   |  |  +-- babel-txcost
+   |  |  +-- babel-exp-mcast-hello-seqno
+   |  |  +-- babel-exp-ucast-hello-seqno
+   |  |  +-- babel-ucast-hello-seqno
+   |  |  +-- babel-ucast-hello-interval
+   |  |  +-- babel-rxcost
+   |  |  +-- babel-cost
+   +-- babel-routes
+   |  +-- babel-route-prefix
+   |  +-- babel-route-prefix-length
+   |  +-- babel-route-router-id
+   |  +-- babel-route-neighbor
+   |  +-- babel-route-received-metric
+   |  +-- babel-route-calculated-metric
+   |  +-- babel-route-seqno
+   |  +-- babel-route-next-hop
+   |  +-- babel-route-feasible
+   |  +-- babel-route-selected
+   +-- babel-security
+      +-- babel-security-mechanism
+      +-- babel-security-enable
+ +++++ need new stuff here

```

Comment:
Interesting how you did this. In YANG we have tools that generate this tree.

Reply:
Yeah, no tools for non-YANG models. And probably not worth the effort to create a tool. I think it's ok. I'll be double-checking to make sure I've got the same items in the summary and the objects and the definitions of object parameters.